cli/command/trust/sign_test.go

```diff
@@ -289,3 +161,15 @@ func TestChangeList(t *testing.T) {
 	cl, err := notaryRepo.GetChangelist()
 	assert.Equal(t, len(cl.List()), 0)
 }
+
+func TestInitNotaryRepoWithSigners(t *testing.T) {
```

Comment:
can we change this test name to reflect the offline nature for now?  `TestInitOfflineNotaryRepoWithSigners` or similar?